src/main/resources/i18n/messages.properties

```diff
@@ -997,6 +997,8 @@ teammatchmaking.queueTitle.inParty = The party leader will join the queues
 teammatchmaking.queueTitle.inQueue = Your party is now in queue
 teammatchmaking.queue.ladder1v1 = 1v1 Queue
 teammatchmaking.queue.tmm2v2 = 2v2 Queue
+teammatchmaking.queue.tmm4v4ShareUntilDeath = 4v4 Casual
+teammatchmaking.queue.tmm4v4FullShare = 4v4 Competitive
```

Comment:
Is that really what we're going to make the technical names of the queues? Wouldn't it make sens to have the technical names also just be `tmm4v4_casual` and `tmm4v4_comp`?

Reply:
api values should be camelCase 😝

Reply:
Now that I think about it, I guess you are right. I wanted to include what the actual difference is, but in the db where the nameKey is defined the gameoptions are right next to it, so that is probably not needed.

Reply:
Well I was just referencing the technical names as they'd appear in the db and in the server protocol (they should be snake case). I assume the API will transform those but idk.